src/modules/project/export.ts

```diff
@@ -345,21 +345,20 @@ export async function downloadFiles(args: {
   }
 
   // Download models
-  total += Object.keys(mappings).length
+  const paths = Object.keys(mappings).filter(path => (isDeploy ? !path.endsWith('.ts') : !path.endsWith('.js')))
```

Comment:
I moved this thingy up here to fix a minor issue where the progress sometimes would not reach 100% because of the mappings filtered out. With this line moved up here now the progress bar always reaches 100%.